src/server/middleware/error.middleware.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response, NextFunction } from 'express';
+import { CustomError, CustomErrors, errorTypeToStatusCode } from '../custom-error.model';
+
+export const isCustomError = (error: Error | CustomError) => error instanceof CustomError;
```

Comment:
whats the point of this function?

Reply:
i wrote this in my project, so richie won't know, he just copied